src/mono/wasm/build/WasmApp.Native.targets

```diff
@@ -194,7 +190,7 @@
 
       <_EmccCommonFlags Include="$(_DefaultEmccFlags)" />
       <_EmccCommonFlags Include="$(EmccFlags)" />
-      <_EmccCommonFlags Include="-s EXPORT_ES6=1"                   Condition="'$(WasmEnableES6)' != 'false'" />
+      <_EmccCommonFlags Include="-s EXPORT_ES6=1" />
```

Comment:
I think this flag needs to be moved to be just for compiling? I saw emcc warnings that it is not relevant or used when linking. Doesn't block this PR.

Reply:
I created follow-up issue for this https://github.com/dotnet/runtime/issues/72990